test/keyv.js

```diff
@@ -75,5 +75,22 @@ test.serial('.set(key, val, ttl) where ttl is "0" overwrites default tll option
 	tk.reset();
 });
 
+test.serial('Keyv uses custom serializer when provided instead of json-buffer', async t => {
+	t.plan(2);
+	const serialize = obj => {
+		t.pass();
+		return JSON.stringify(obj);
+	};
+	const deserialize = str => {
+		t.pass();
+		const obj = JSON.parse(str);
+		return obj;
+	};
+	const store = new Map();
+	const keyv = new Keyv({ store, serialize, deserialize });
+	await keyv.set('foo', 'foo');
+	await keyv.get('foo');
```

Comment:
Test the output to make sure everything was definitely saved `t.is(await keyv.get('foo'), 'bar');`

Will require updating `t.plan()`.